frontend/public/components/factory/table.tsx

```diff
@@ -172,6 +173,14 @@ const sorts = {
   volumeSnapshotSource: (snapshot: VolumeSnapshotKind): string => snapshotSource(snapshot),
   snapshotLastRestore: (snapshot: K8sResourceKind, { restores }) =>
     restores[getName(snapshot)]?.status?.restoreTime,
+  sortPackageManifest: (packageManifest: PackageManifestKind): string => {
```

Comment:
See earlier comment about a more descriptive name for this. Maybe something like `sortPackageManifestByDefaultChannelName`. Also this logic is repeated in `PackageManifestTableRow` component, which means it is a good candidate for  a utility function. There is already a `defaultChannelFor` function in `./packages/operator-lifecycle-manager/src/components/index.tsx`, but it looks like that is essentially just a basic accessor for a channel name. You could rename that one to `defaultChannelNameFor` and make `defaultChannelFor` return the whole channel object.